charts/images.yaml

```diff
@@ -87,7 +87,21 @@ images:
   sourceRepository: github.com/kubernetes/cloud-provider-openstack
   repository: registry.k8s.io/provider-os/openstack-cloud-controller-manager
   tag: "v1.26.2"
-  targetVersion: ">= 1.26"
+  targetVersion: ">= 1.26.x"
+  labels:
+  - name: 'gardener.cloud/cve-categorisation'
+    value:
+      network_exposure: 'protected'
+      authentication_enforced: false
+      user_interaction: 'gardener-operator'
+      confidentiality_requirement: 'high'
+      integrity_requirement: 'high'
+      availability_requirement: 'low'
+- name: cloud-controller-manager
+  sourceRepository: github.com/kubernetes/cloud-provider-openstack
+  repository: registry.k8s.io/provider-os/openstack-cloud-controller-manager
+  tag: "v1.27.1"
+  targetVersion: ">= 1.27"
   labels:
   - name: 'gardener.cloud/cve-categorisation'
```

Comment:
You have to update `registry.k8s.io/provider-os/cinder-csi-plugin:v1.27.1` and `registry.k8s.io/provider-os/manila-csi-plugin:v1.27.1` images to be used for K8s 1.27 Shoots.